Workbooks/Storage/Overview/Overview.workbook

```diff
@@ -344,6 +344,13 @@
               "showIcon": true
             }
           },
+          {
+            "columnMatch": "success/Transactions",
```

Comment:
Merge this with the setting that hides the common 'Success/Transactions' using an OR regex